File: js/al4/3.js


function Factorial(number){
    let factorial = 1;
    for (let i = 1; i <= number; i++){
        factorial = factorial * i;
    }
    return factorial;
}

let x = 2;
let sum = 0;
let n = 12;

let max = 2 * n - 1;

for (j = 3; j <= max; j+= 2 ){
    sum += x ** j / Factorial(j);
}

console.log(x - sum);

File: python /al4/3.py
def factorial(n):
    if n == 0:
        return 1
    else:
        return n * factorial(n - 1)

x = 2
sum = 0
n = 12

max_val = 2 * n - 1

for j in range(3, max_val + 1, 2):
    sum += x ** j / factorial(j)

result = x - sum
print(result)